pennylane/ops/op_math/condition.py

```diff
@@ -222,45 +223,47 @@ def __call_capture_enabled(self, *args, **kwargs):
             else self.orig_elifs
         )
 
-        @wraps(self.true_fn)
-        def new_wrapper(*args, **kwargs):

```

Comment:
Why are we no longer returning a wrapper fn?

Reply:
The wrapper function was merely an accidental byproduct of merging the capture implementation and the default implementation into the same class. It's unnecessary complexity and indirection.